### PR TITLE
Add hard-coded comments data

### PR DESCRIPTION
**Pull request by ankitamitra**
From the walkthrough spec (step 3): 
"At this point, your server should be returning hard-coded comments data, and your JavaScript should be fetching this data and using it to add HTML elements to the page."

*Deployed but takes a few seconds to load

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,39 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (saur-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:24]: do we need to update this comment

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,39 @@
...
+    String json = convertToJson(messages);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+  /**
+   * Converts an Arraylist<String> instance into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(ArrayList<String> messages) {
```
Comment (saur-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:45]: Looks like, you already added dependency for gson, do you want to use that for serializing response. see: https://stackoverflow.com/questions/27893342/how-to-convert-list-to-a-json-object-using-gson

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,39 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```
Comment (saur-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: Do you want to use https://github.com/google/google-java-format to format the code (e.g. sort the imports).

Replace the wildcard imports with actual imports in use: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (ankitamitra) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Makes sense! Fascinatingly, gson Unicode escapes certain characters, of which the apostrophe is one (https://stackoverflow.com/questions/11361176/can-someone-clarify-gsons-unicode-encoding). The only (easy-ish) solution seems to be to use disableHtmlEscaping(), but that makes it slightly less clean. 

Edited the hard-coded comments for now, but will work on figuring out the disableHtmlEscaping() soon!

**Pull request merged**